src/Main.java

```diff
@@ -1,9 +1,82 @@
+import java.util.Scanner;
+import java.util.HashMap;
+import java.util.Map;
 
 public class Main {
     public static void main(String[] args) {
 
-        // Hier mag je je code schrijven voor de hoofd-opdracht
+        // Create numeric array
+        int[] numericNumbers = new int[10];
+        for (int i = 0; i < 10; i++) {
+            numericNumbers[i] = i;
+        }
```

Comment:
Dit werkt prima (en werkt bij grotere reeksen misschien zelfs beter) maar in dit geval had je het ook zonder loop kunnen doen en gewoon {1,2,3 etc..}